core/src/test/scala/dagr/core/execsystem/TaskManagerTest.scala

```diff
@@ -44,18 +44,22 @@ object TaskManagerTest {
   }
 }
 
-class TaskManagerTest extends UnitSpec with PrivateMethodTester with OptionValues with LazyLogging with BeforeAndAfterAll {
+trait TestTaskManager extends TaskManager {
+  override def joinOnRunningTasks(millis: Long) = super.joinOnRunningTasks(millis)
```

Comment:
Silly stuff since I can’t override a method that has `protected[scope]` outside of that package scope.